src/test/java/TestParser.java

```diff
@@ -0,0 +1,32 @@
+import duke.Parser;
+import duke.exception.DukeException;
```

Comment:
Perhaps you might want to consider leaving a line between the two? To be consistent with FindCommand.java